### PR TITLE
ledger-live-desktop: 2.1.0 -> 2.3.2

### PR DESCRIPTION
**Pull request by nyanloutre**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by pnelson** [comment]
Builds and runs fine.

**Review by pnelson — APPROVED** [review]
Builds and runs fine.

File: pkgs/applications/blockchains/ledger-live-desktop/default.nix [hunk 1]
```diff
@@ -2,12 +2,12 @@
 
 let
   pname = "ledger-live-desktop";
-  version = "2.1.0";
+  version = "2.2.4";
```
Comment (FelschR) [inline pkgs/applications/blockchains/ledger-live-desktop/default.nix:5]:
```suggestion
  version = "2.3.2";
```

File: pkgs/applications/blockchains/ledger-live-desktop/default.nix [hunk 2]
```diff
@@ -2,12 +2,12 @@
...
   name = "${pname}-${version}";
 
   src = fetchurl {
     url = "https://github.com/LedgerHQ/${pname}/releases/download/v${version}/${pname}-${version}-linux-x86_64.AppImage";
-    sha256 = "1ywvdqmq8asczhmvc6ai2v6di1f5q19x3ygqlinwz8d1hrj3496r";
+    sha256 = "0ymn7jp14jaj3v4yj32kyxic47qa87rbngkdld1p977g7zg2pc86";
```
Comment (FelschR) [inline pkgs/applications/blockchains/ledger-live-desktop/default.nix:10]:
```suggestion
    sha256 = "0xd3w564zij614ajg57n1qlvz0hd9l7219qxx81ai6b02b9a5g9h";
```

**Review by felschr — COMMENTED** [review]
v2.3.2 has been released.

**Comment by FelschR** [comment]
Installs and runs fine for me.

**Comment by bhipple** [comment]
Result of `nixpkgs-review pr 85646` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
<br>- ledger-live-desktop
</details>

**Review by bhipple — APPROVED** [review]

**Pull request merged**